src/current/v24.1/automatic-cpu-profiler.md

```diff
@@ -55,19 +41,9 @@ The Automatic CPU Profiler uses the configuration options to determine the high-
 
 Enabling the automatic CPU profile capture on a cluster will add overhead to the cluster in the form of potential increases in latency and CPU usage.
 
-{{site.data.alerts.callout_info}}
-The decision to enable this feature should be done when advised by the [Cockroach Labs support team]({% link {{ page.version.version }}/support-resources.md %}).
-{{site.data.alerts.end}}
-
 - Monitor the following metrics:
   - [P99 latency]({% link {{ page.version.version }}/ui-sql-dashboard.md %}#service-latency-sql-99th-percentile)
   - P50 latency by creating a [custom chart]({% link {{ page.version.version }}/ui-custom-chart-debug-page.md %}) for the `sql.exec.latency-p50` metric
   - [CPU usage]({% link {{ page.version.version }}/ui-hardware-dashboard.md %}#cpu-percent)
-- We anticipate a sub-10% regression on these foreground latency metrics. This overhead to your cluster may be deemed acceptable in order to collect CPU profiles that are necessary to troubleshoot problems in your cluster. Please consult with Cockroach Labs Support.
-- Overhead only occurs during profile capture, not when it is idle. If the cluster can tolerate a `server.cpu_profile.duration` (for example, 1 second) increase in latency to capture the CPU profile, consider enabling the Automatic CPU Profiler.
-
-{{site.data.alerts.callout_danger}}
-Do not have the Automatic CPU Profiler enabled by default.
-
-If you enabled the Automatic CPU Profiler and then notice unacceptable overhead to your cluster, we recommend you immediately disable the Automatic CPU Profiler.
-{{site.data.alerts.end}}
+- We anticipate a sub-10% regression on these foreground latency metrics. This overhead to your cluster may be deemed acceptable in order to collect CPU profiles that are necessary to troubleshoot problems in your cluster.
```

Comment:
Let's remove this line and the following one completely. I went through the [PR](https://github.com/cockroachdb/cockroach/pull/95623) that implemented the CPU profiler and don't seen any reason why we state the regression. Testing in the PR showed no noticeable "regression".

Reply:
removed 2 lines